frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/CustomReceiptContent/RedirectToCreatePageButton/RedirectToCreatePageButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classes from './RedirectToCreatePageButton.module.css';
+import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
+import { PackagesRouter } from 'app-shared/navigation/PackagesRouter';
+import { PencilWritingIcon } from '@studio/icons';
+import { StudioButton, StudioLabelAsParagraph } from '@studio/components';
+import { useTranslation } from 'react-i18next';
+
+export const RedirectToCreatePageButton = (): React.ReactElement => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioUrlParams();
+  const packagesRouter = new PackagesRouter({ org, app });
+
+  return (
+    <div className={classes.goToCreatePageWrapper}>
+      <StudioLabelAsParagraph size='small'>
+        {t('process_editor.configuration_panel_custom_receipt_navigate_to_lage_title')}
+      </StudioLabelAsParagraph>
+      <div>
+        <StudioButton
+          as='a'
+          size='small'
+          variant='primary'
+          color='second'
+          icon={<PencilWritingIcon />}
+          href={packagesRouter.getPackageNavigationUrl('editorUiEditor')}
```

Comment:
I think this does not work if adding another (data) task after adding the receipt. Since the layoutset is set to the selected one in the addLayoutset mutation. Actually I think we should remove setting the state from that hook and just let the processeditor not affect the layoutset in Lage at all, except when deleting. But in this particular case we can set the selectedLayoutSet directly when clicking this button, maybe?

Reply:
Fixed 😄